pkg/kubelet/cm/topologymanager/socketmask/socketmask_test.go

```diff
@@ -137,6 +137,36 @@ func TestOr(t *testing.T) {
 	}
 }
 
+func TestXOr(t *testing.T) {
+	tcases := []struct {
+		name          string
+		firstMaskBit  int
+		secondMaskBit int
+		orMask        string
```

Comment:
```suggestion
		xorMask        string
```

Can we call this `xorMask` instead? I think its clearer.